<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShapeOrientationDialogBase</class>
 <widget class="QDialog" name="ShapeOrientationDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shape Orientation Plugin</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>0</y>
     <width>416</width>
     <height>221</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Polygons">
    <attribute name="title">
     <string>Use polygon layer</string>
    </attribute>
    <widget class="QLabel" name="label_PolyShp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Layer name</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbox_PolyShp">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>231</width>
       <height>36</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_1">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>71</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>100</y>
       <width>131</width>
       <height>36</height>
      </rect>
     </property>
     <property name="minimum">
      <double>0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_polyInter">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>100</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Sectors angles (degrees)</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbox_PolyIdField">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>55</y>
       <width>231</width>
       <height>36</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_PolyIdField">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>65</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Id Field</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Poly">
     <property name="geometry">
      <rect>
       <x>305</x>
       <y>140</y>
       <width>92</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>Begin</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Lines">
    <attribute name="title">
     <string>Use line layer</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Layer name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>71</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbox_LineShp">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>231</width>
       <height>36</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>105</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Sectors angles (degrees)</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>100</y>
       <width>131</width>
       <height>36</height>
      </rect>
     </property>
     <property name="minimum">
      <double>0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_LineIdField">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>65</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Id Field</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbox_LineIdField">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>55</y>
       <width>231</width>
       <height>36</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Line">
     <property name="geometry">
      <rect>
       <x>305</x>
       <y>140</y>
       <width>92</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>Begin</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_Output">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>295</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Output folder</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_Output">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Output">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>290</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkDiagram">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Plot rose diagram</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cbox_ColorRamp">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>156</width>
     <height>36</height>
    </rect>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_ColorRamp">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Plot colour scheme</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
